Userland/Applications/PixelPaint/Image.cpp

```diff
@@ -392,13 +392,13 @@ void Image::merge_visible_layers()
 
 void Image::merge_active_layer_up(Layer& layer)
 {
-    if (m_layers.size() < 2)
-        return;
+    // NOTE: The action for this should be disabled if there are fewer than 2 layers.
+    VERIFY(m_layers.size() >= 2);
```

Comment:
This is completely unrelated to the PR, but I was wondering if it makes sense to add an extra parameter to `VERIFY` which can be used for a reason/justification so you are encouraged to leave notes (like the one above this)